backend/server.go

```diff
@@ -182,63 +192,58 @@ var roleCast = map[string]gamelogic.Role{
 	"doppleganger": gamelogic.DoppleGanger,
 }
 
-func AssignRolePool(game *gamelogic.Game, responseWriter http.ResponseWriter, request *http.Request) (error, int) {
+func AssignRolePool(game *gamelogic.Game, responseWriter http.ResponseWriter, request *http.Request) (JsonBody, HttpStatus, error) {
 	rolesReq := struct{ Roles []string }{}
 
 	if err := json.NewDecoder(request.Body).Decode(&rolesReq); err != nil {
-		return InvalidJSON, http.StatusBadRequest
+		return nil, http.StatusBadRequest, InvalidJSON
 	}
 
 	var roles []gamelogic.Role
 
 	for _, roleName := range rolesReq.Roles {
 		role, ok := gamelogic.RoleNameToID[roleName]
 		if !ok {
-			return RoleNoteFoundError, http.StatusBadRequest
+			return nil, http.StatusBadRequest, RoleNoteFoundError
 		}
 		roles = append(roles, role)
 	}
 
 	game.AssignRolePool(roles)
-	fmt.Fprint(responseWriter, "{}")
 
-	return nil, http.StatusOK
+	return emptyJson, http.StatusOK, nil
```

Comment:
I said this before we should allow this to return nil and the wrap endpoint should just construct an empty json for us.

Reply:
yeah i like that better